des/DES.py

```diff
@@ -4,76 +4,114 @@
 from car import Car
 import time
 
-# cp.process()
-# cp.timeout()
-# cp.run()
-# cp.now
-## Resources (such as cp and parking lots)
-## Resource() : limit num of processes, need request --> parking lot
-## Container() :  discrete/continuous quantities
-## Store() : unlimited --> carpark
+## time unit : minutes
 
 ## TODO: take input from user / database
-# 1h = 3600s
-# 1D = 86400s
-# 1 week = 604800
-SIM_TIME = 86400 # in seconds
-ARRIVAL_RATE = 1
-cp_info = {'cp3' : 236, 
-           'cp3a' : 41, 
-           'cp4' : 95, 
-           'cp5' : 70, 
-           'cp5b' : 24, 
-           'cp6b' : 163, 
-           'cp10' : 232}
-cp_prob = [0.2, 0.2, 0.05, 0.05, 0.1, 0.15, 0.25]
-car_types = ['hourly', 'student', 'staff', 'esp']
-car_prob = [0.35, 0.1, 0.5, 0.05]
-
-def custom_choice(list, prob):
+SIM_TIME = 24 * 60 # in minutes
+ARRIVAL_RATE = 60 # n : 1 car every n minute
+
+CP_CAPACITY = {
+    'cp3' : (3, 1), 
+    # 'cp3a' : (41, 0), 
+    # 'cp4' : (95, 0), 
+    # 'cp5' : (70, 0), 
+    # 'cp5b' : (24, 0), 
+    # 'cp6b' : (163, 0), 
+    # 'cp10' : (232, 0)
+}
+
+CP_PROB = {
+    'cp3' : 1 # 0.2, 
+    # 'cp3a' : 0.2, 
+    # 'cp4' : 0.05, 
+    # 'cp5' : 0.05, 
+    # 'cp5b' : 0.1, 
+    # 'cp6b' : 0.15, 
+    # 'cp10' : 0.25
+}
+
+CAR_PROB = {
+    'hourly' : 0.35, 
+    'student' : 0.1, 
+    'staff' : 0.5, 
+    'esp' : 0.05
+}
+
+def custom_choice(items, prob):
     rng = np.random.default_rng()
-    return rng.choice(list, p=prob)
+    return rng.choice(items, p=prob)
+
+def create_cp(env, cp_dict):
```

Comment:
Nit: might be clearer to declare the typings of params and docstrings

Reply:
will do